library/spdm_responder_lib/libspdm_rsp_finish.c

```diff
@@ -431,9 +431,9 @@ libspdm_return_t libspdm_get_response_finish(libspdm_context_t *spdm_context, si
     }
 
     if (((session_info->mut_auth_requested == 0) &&
-         (spdm_request->header.param1 != 0)) ||
+         ((spdm_request->header.param1 & 0x1) != 0)) ||
```

Comment:
would you please `SPDM_FINISH_REQUEST_ATTRIBUTES_SIGNATURE_INCLUDED` to replace the hardcode value?

Reply:
I have changed the code. Thanks.